### DISCUSSION TITLE
Wondering if WebViews for PayPal and Klarna are included in the SDK?

### DISCUSSION THREAD

**Discussion by divyegd** [Adyen/adyen-react-native#481] [Q&A] [answered]
We are implementing pay with PayPal and Klarna in the mobile app. On submit payment request, a URL is returned which we have to load in a WebView. 

We are wondering if there is a way to trigger the WebView using this SDK?

**Comment by descorp** [comment, marked as answer]
Hey @divyegd 

> if there is a way to trigger the WebView using this SDK

Yes there is :)

If you are [using Drop-in](https://docs.adyen.com/online-payments/build-your-integration/sessions-flow/?platform=Cross-platform&integration=React+Native+Drop-in&version=2.0.0) or [Components](https://docs.adyen.com/online-payments/build-your-integration/advanced-flow/?platform=Cross-platform&integration=React+Native+Components&version=2.0.0) - it is part of the flow.

In case you are not using our pre-build payment UI - we provide standalone action handling (See [example](https://github.com/Adyen/adyen-react-native/blob/develop/example/src/Utilities/payWithCard.js#L29))
